fs/expand_glob.ts

```diff
@@ -316,12 +311,8 @@ export async function* expandGlob(
       return;
     } else if (globSegment === "..") {
       const parentPath = joinGlobs([walkInfo.path, ".."], globOptions);
-      try {
-        if (shouldInclude(parentPath)) {
-          return yield await createWalkEntry(parentPath);
-        }
-      } catch (error) {
-        throwUnlessNotFound(error);
+      if (shouldInclude(parentPath)) {
+        return yield await createWalkEntry(parentPath);
```

Comment:
Can you please explain this change?

Reply:
This try-catch block tries to handle `Deno.errors.NotFound` thrown from `Deno.stat` (called in `createWalkEntry`), and re-throw other errors as is (see `throwUnlessNotFound` for details).

However, `Deno.errors.NotFound` is not possible to  be thrown here because the target path (`parentPath`) is created by `joinGlobs([walkInfo.path, ".."])` which is parent path of existing path (`walkInfo` is only created for existing path). (e.g. If `foo/bar/baz` exists, `foo/bar` needs to exist.)

If `Deno.errors.NotFound` is not possible to be thrown, this try-catch block does nothing.

Reply:
Cool. As I thought. Thanks!